scheduler/kubernetes/snaps-3d.md

```diff
@@ -44,62 +58,127 @@ spec:
       value: mysql --user=root --password=$MYSQL_ROOT_PASSWORD -Bse 'flush tables with read lock;system ${WAIT_CMD};'
 ```
 
-Below rule flushes the tables from the memtable on all cassandra pods.
+**Snapshot**
+
+Creating the below VolumeSnapshot will do the following:
+
+* Stork will run the _px-presnap-rule_ rule on all pods that are using PVCs that match labels _app=mysql_.
+* Once the rule is executed, Stork will take a snapshot of all PVCs that match labels _app=mysql_. Hence this will be a group snapshot.
```

Comment:
This is confusing, mysql itself will only have one volume, so don't need a group snapshot. Cassandra below is good example of group snap, so we can remove the selector for mysql.

Reply:
done